lib/extensions/ar_migration.rb

```diff
@@ -15,7 +15,10 @@ def self.update_local_migrations_ran(version, direction)
     new_migrations = ActiveRecord::SchemaMigration.normalized_versions
     new_migrations << version if direction == :up
 
-    (new_migrations - schema_migrations_ran_class.pluck(:version)).each do |v|
+    to_add = (new_migrations - schema_migrations_ran_class.pluck(:version))
+    puts "Seeding :schema_migrations_ran table..." if to_add.size > 1
```

Comment:
Why the colon here? Shouldn't it just be the table name?

Reply:
I can also put a leading `==` stuff around it to for even more visual consistency, if you think that's better.

Reply:
I really wanted to use say_with_time, but we are not in the same context as that method.